app/views/inventory_reconciliations/_incomplete_deltas.html.erb

```diff
@@ -19,17 +19,19 @@
 
       <tbody>
         <% @reconciliation.deltas.each do |delta| %>
-          <tr class="<%= delta.row_css_class %>" <%= delta.warning_path %> <%= delta.warning_text %>>
-            <td class="<%= delta.description_css_class %>"><%= delta.item.category.description %> - <%= delta.item.description %></td>
+          <% reconciliation_delta_table_row(delta) do %>
```

Comment:
This is missing an `=`

`<%= reconciliation_delta_table_row(delta) do %>`

All rendering rails inserts must include the `=` in the declaration